source/docs/hardware-components/fasteners-guide.rst

```diff
@@ -0,0 +1,182 @@
+..    include:: <isonum.txt>
+
+Fasteners Guide
```

Comment:
```suggestion
Fastener Guide
```
I think this title fits better with other gm0 pages (the file should also be renamed to `fastener-guide.rst`